php/libraries/User.class.inc

```diff
@@ -66,26 +66,41 @@ class User extends UserPermissions
         $row = $DB->pselectRow($query, array('UID' => $username));
```

Comment:
The function is called using `$username` so I don't think it's possible for it to be unset